tests/drivers/spi/spi_loopback/src/spi.c

```diff
@@ -33,6 +33,8 @@ struct spi_cs_control spi_cs = {
 #define CS_CTRL_GPIO_DRV_NAME ""
 #endif
 
+/* connect MOSI pin and the MISO of the SPI */
```

Comment:
Your commit message is misleading, it seems it just adds exception when stm32 dma is in use.

Also, this comment, and all the log message changes are not related to this commit, so have a separate patch.

Reply:
This is now splitted in 2 commits :
1. b972c68c0a89041c84eb18ba20a7fab32d2a8949 is for adding more comments to the test applications 

2. e75fc5ee50904ffec513330f637526c1d4194ba3 is to adapt for dma transfer